content/blog/deploying-with-octopus-and-pulumi/index.md

```diff
@@ -0,0 +1,26 @@
+
+[Octopus Deploy](https://octopus.com) is a deployment automation server that integrates with your existing build pipeline or through a REST API. You can select the environments for deploying your software, and who has rights to complete a deployment. For example, the QA team can deploy to the Test environment but not to Development or Production. Different team members may trigger deployments, and the deployment process is always consistent.
+
+## Continuous Delivery with Octopus Deploy
+
+Deployment targets are grouped in **environments** in Octopus. Environments represent different stages of the deployment pipeline that your software passes through on its way to a release. A common practice is to create unique Development, Test, and Production environments. Octopus Deploy lets you deploy the correct versions of your software, with the right configuration for the appropriate environment.
+
+Pulumi's [guide](https://www.pulumi.com/docs/guides/continuous-delivery/octopus-deploy/) to implementing Octopus Deploy walks you through the process of implementing an environment for deploying a Python flask application on [AWS Fargate](https://aws.amazon.com/fargate/). Before starting, make sure you have met the [prerequisites](https://www.pulumi.com/docs/guides/continuous-delivery/octopus-deploy/#prerequisites).
+
+Octopus environments are analogous to Pulumi stacks. If you have Development, Test, and Production Octopus environments, you would create a Pulumi stack for each environment. The application being deployed is a Pulumi project. Octopus uses packages that contain the source code bundled in a supported format such as .zip files, gzip or bzip tar files,  Java archive formats such as .jar or .war, and Docker images. Pulumi apps can be packaged in the desired format and extracted onto a Octopus worker that the Pulumi CLI can access.
+
+In Octopus, a **Process** is made up of the steps needed to execute a project. To deploy a Pulumi package, additional steps are needed to extract the code. You can add a step to a process by using the Octopus menu. In the Process section, select **AddStep** to add the package. You will need to specify the package feed, the package id and a package name. The step can be further customized by adding a script that specifies where to extract the package.
+
+To deploy your Pulumi package, select **Add Step** again and search for the `Pulumi` template from the **Community Library Steps**. Fill out the template with the appropriate configuration parameters for your stack. Next, create a release from by selecting **Releases** and click **Create Release** and follow the instructions.
```

Comment:
I think we should highlight or maybe include a screenshot of the [Pulumi Step Template](https://library.octopus.com/step-templates/76296cd1-7d8c-47e8-b33f-027ecd3ff6b5/actiontemplate-run-pulumi-(linux)) from the [Community Library](https://library.octopus.com). Basically, some sort of an introduction that we have created a free step template to help with deploying Pulumi apps on Octopus Deploy, followed by this paragraph that says how you can use it.

Reply:
Added screenshot